samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md

```diff
@@ -1,3 +1,7 @@
 # Role based access sample with Microsoft.Azure.EventHubs SDK #
 
 For more information on Role based access (RBAC) and how to run this sample follow [this](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-role-based-access-control) link.
+
+**Note**: This sample uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the new library `Azure.Messaging.EventHubs`. 
```

Comment:
nit:  maybe instead of "new" we go with something like "current generation" or "current"?    New is only "new" for so long.... I would think we want to position this as "the current thing that we should use".